src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -205,7 +205,7 @@ private static async void ReloadWithUpdatedTypes(Type[] updatedTypes)
 
 				var isCapturingState = true;
 				var treeIterator = EnumerateHotReloadInstances(
-						Window.Current.Content,
+						Windows.UI.Xaml.Window.Current!.Content!,
```

Comment:
This needs to be adjusted for multiwindow support, ideally in a separate PR - @nickrandolph @jeromelaban - it would be possible to for example enumerate over `ContentRoots` and take the `VisualTree.RootElement` of each

Reply:
Adjusted the code to run for all `Windows` (not content roots, so for now it will ignore Uno Islands)